lib/npm/test.js

```diff
@@ -28,6 +28,10 @@ function authorName(author) {
 }
 
 function test(context, next) {
+  if (context.options.script) {
+    // This handles the citgm.js case
+    context.module.script = context.options.script;
```

Comment:
okay so citgm.js puts script in `context.options.script` and citgm-all.js put script in `context-module-script` This was broken in a recent PR but I couldn't find which one